src/videojs.ga.coffee

```diff
@@ -66,11 +68,13 @@ videojs.plugin 'ga', (options = {}) ->
         seeking = true
         sendbeacon( 'seek start', false, seekStart )
         sendbeacon( 'seek end', false, seekEnd )
+        percentsAlreadyTracked = [];
```

Comment:
when you seek not sure to say we can say that you read the previous percent of the video